features/product/managing_products/sorting_products.feature

```diff
@@ -63,3 +64,22 @@ Feature: Sorting listed products
         And I sort the products descending by name
         Then I should see 3 products in the list
         And the first product on the list should have name "Szałowy Mops"
+
+    @ui
+    Scenario: Missing translations are sorted as first when sorting by name ascending
+        When I change my locale to "Polish"
+        And I browse products
+        And I sort the products ascending by name
+        Then I should see 3 products in the list
+        And the first product on the list should have "data-test-missing-translation-paragraph" data attribute
```

Comment:
One of the rules of rightful BDD is to omit implementation details in the steps. I wouldn't expect anyone non-tech to say a sentence like this in real life. Also, such a step force us to interpret it only in an HTML context. 
```suggestion
        And the first product on the list shouldn't have a name
```

Reply:
I've implemented you suggestion but due to some reason 3 actions are failing and they seem to not be related to my changes at all. Can you take a look? Maybe something like this happened earlier? The error messages doesn't tell me too much 🤷🏻‍♀️.